scene/gui/control.cpp

```diff
@@ -1373,6 +1377,10 @@ void Control::_size_changed() {
 	}
 }
 
+void Control::_tree_exiting() {
+	release_focus();
+}
```

Comment:
Wouldn't be better to add this to the `NOTIFICATION_EXIT_TREE` branch of the `_notification()` function instead of creating another one?

Reply:
Yes this is the better option. I am rather new to contributing to godot, so I used a signal.